src/colejorz/pilothouse.py

```diff
@@ -36,7 +39,7 @@ class Pilothouse:  # pylint:disable=too-many-instance-attributes
     STOP = 'stop'
     BACKWARD = 'backward'
 
-    def __init__(self, queue):
+    def __init__(self, queue: Queue) -> None:
```

Comment:
Would the type of items in the queue be useful here? This might need the `"Queue[...]"` notation if `Queue[...]` raises a runtime error.